html/resources/files/tagline-off-android-v3.json

```diff
@@ -7,141 +7,244 @@
       "translations": {
```

Comment:
@g123k where had the Nutri-Score v2 campaign gone by the way ? Did we ever publish it for the rest of the world ?